css/terminal7.css

```diff
@@ -346,14 +346,11 @@ button.rename-close {
 .zoomed .pane {
     height: -webkit-fill-available;
 }
-#map:target {
```

Comment:
I'm testing your branch and I'm getting a bit of the map surrounding the panes. Is that intentional?

![IMG_1719](https://user-images.githubusercontent.com/36852/192738771-a0d0ee0c-8bb3-49e3-af69-9cc2e5e794ce.jpeg)

Reply:
I've restored the lines and merged the PR. I haven't pushed the merged master yet as I've been testing it. 

Moving forward, this PR minus the css changes will be pushed later today. Please open a bug with a detailed description on how to recreate the problem. There we can discuss how to fix navigation then you'll know what to work on.

It'll be best if you start with a `./qa/ui/ui.spec.ts` test to recreate the bug. Once we have a failing test it's much faster to develop and test as we don't need a browser. What's more, it's the only way to ensure the bug will never come back.

Reply:
I'm almost sure this bug is already fixed.